src/nft/components/details/detailsV2/LandingPage.tsx

```diff
@@ -0,0 +1,100 @@
+import Column from 'components/Column'
+import Row from 'components/Row'
+import { VerifiedIcon } from 'nft/components/icons'
+import { CollectionInfoForAsset, GenieAsset } from 'nft/types'
+import styled from 'styled-components/macro'
+import { BREAKPOINTS, ThemedText } from 'theme'
+
+const MAX_WIDTH = 560
+
+const Container = styled.div`
```

Comment:
nit: could use a styled(Column) to simplify styles

Reply:
nit: rename `LandingPageContainer` to be more explicit

Reply:
considering the flex direction changes i don't know if that is the best